app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    done_lst_path = command.split()
```

Comment:
Code Efficiency: The `command.split()` function splits the string into a list where each word is a list item. However, it does not check if the `cp` command is correct. You should add a check to ensure that the first item of `done_lst_path` is 'cp'.